cumulus/bin/pov-validator/Cargo.toml

```diff
@@ -19,8 +19,8 @@ sc-executor.workspace = true
 sp-core.workspace = true
 sp-io.workspace = true
 sp-maybe-compressed-blob.workspace = true
-tracing-subscriber.workspace = true
 tracing.workspace = true
```

Comment:
was the toml format test failing before you touched this file?
It should only validate modified files from the PR

Reply:
Yes, even when I revert the changes I get the check errors (Locally).
It was formatted  I ran the `taplo format --config .config/taplo.toml` command.

Reply:
Let's try reverting these changes. They shouldn't fail in the CI, as they wouldn't have been merged otherwise in the first place.